code/game/objects/items/storage/boxes.dm

```diff
@@ -226,13 +226,13 @@
 	new /obj/item/reagent_containers/glass/beaker/noreact(src)
 	new /obj/item/reagent_containers/glass/beaker/bluespace(src)
 
-/obj/item/storage/box/medsprays
-	name = "box of medical sprayers"
-	desc = "A box full of medical sprayers, with unscrewable caps and precision spray heads."
+/obj/item/storage/box/medigels
+	name = "box of medical gels"
+	desc = "A box full of medical gel applicators, with unscrewable caps and precision spray heads."

```

Comment:
>spray heads

Reply:
What do you suggest i call them? The idea is that the gel is sprayed over the wound where it dries into a patch like film.

If i can a good suggestion on how to place every instance of spray in the medigel text i am willing to do so.

Reply:
I don't see an issue, I've got screen-cleaning gel that's sprayed.